src/Features/CSharp/Portable/EditAndContinue/SyntaxUtilities.cs

```diff
@@ -187,7 +187,8 @@ public static SyntaxNode TryGetEffectiveGetterBody(ArrowExpressionClauseSyntax p
                 return propertyBody.Expression;
             }
 
-            return accessorList?.Accessors.Where(a => a.IsKind(SyntaxKind.GetAccessorDeclaration)).FirstOrDefault()?.Body;
+            var firstAccessor = accessorList?.Accessors.Where(a => a.IsKind(SyntaxKind.GetAccessorDeclaration)).FirstOrDefault();
```

Comment:
ok. thanks!

---
In reply to: [128831749](https://github.com/dotnet/roslyn/pull/20931#discussion_r128831749) [](ancestors = 128831749)